packages/labs/ssr/src/lib/module-loader.ts

```diff
@@ -247,6 +247,19 @@ export class ModuleLoader {
     };
     this.cache.set(modulePath, moduleRecord);
     const module = await modulePromise;
+    // Modules must be fully loaded before linking. Therefore `_loadModule` must
```

Comment:
Let's add to the jsdoc comment for _loadModule that this load all dependencies too.